### PR TITLE
224 add prisma to project 1

### PR DESCRIPTION
**Pull request by Elkrival**
Added prisma to project

This commit is an app.js file formatting  and can be skipped
https://github.com/AMP-SCZ/dpdash/commit/7c1eacbf777bffff6e27bf0d4b717dec5868c888

### CODE REVIEW

**Review by mikestone14 — COMMENTED** [review]
This looks good. Can we base this off of the `modernization` branch? Also, are there no queries for sessions in the codebase?

**Comment by Elkrival** [comment]
@mikestone14 That's correct there are no queries for sessions, I think they have it hooked up so that express sessions takes care of that

File: prisma/schema.prisma [hunk 1]
```diff
@@ -6,6 +6,48 @@ generator client {
 }
 
 datasource db {
-  provider = "postgresql"
-  url      = env("DATABASE_URL")
+  provider = "mongodb"
+  url      = env("DPDMONGO_DATABASE_URL")
```
Comment (mikestone14) [inline prisma/schema.prisma:10]: is this env var already set or do we need to set it somewhere?

**Review by mikestone14 — APPROVED** [review]

File: prisma/schema.prisma (hunk 1 above)
Comment (Elkrival) [reply prisma/schema.prisma:10]: it's got to be added to the .env file of the project

**Pull request merged**